help/access/databases.md

```diff
@@ -305,194 +313,241 @@ supported. You can add an input/output of type `list<string>`.
-> without sharing a connection. Deleting a connection also deletes a query.
+<Tabs>
+<TabItem value="patram-query" label="Parameterized query" default>
 
-### Sharing query results as URL
+```sql
+--input: int employeeId = 5
+--input: string shipVia = "=3" {pattern: int}
+--input: double freight = 10.0
+--input: string shipCountry = "France" {choices: Query("SELECT DISTINCT shipCountry FROM Orders")}
+--input: string shipCity = "starts with r" {pattern: string}
+--input: bool freightLess1000 = true
+--input: datetime requiredDate = "1/1/1995"
+--input: string orderDate = "after 1/1/1995" {pattern: datetime}
+SELECT * FROM Orders WHERE (employeeId = @employeeId)
+    AND (freight >= @freight)
+    AND @shipVia(shipVia)
+    AND ((freight < 1000) OR NOT @freightLess1000)
+    AND (shipCountry = @shipCountry)
+    AND @shipCity(shipCity)
+    AND @orderDate(orderDate)
+    AND (requiredDate >= @requiredDate)
+```
 
-To share query results as URL, first, run the query. Once the query is executed, copy the URL from the address bar.
-Query parameters are encoded within the URL. Follow the link to _open_ the dataframe in Datagrok. To access the query
-results from the link provided, users must have permissions to execute this query.
+</TabItem>
+<TabItem value="parameter-dialog" label="Parameter dialog">
+
+![Parametrized queries](parameterized-queries.png)
+
+</TabItem>
+</Tabs>
+
+:::note developers
+
+You can [_call_ a query](../datagrok/functions/function-call.md) across multiple Datagrok plugins instead of copy-pasting the same query every time. To run a query programmatically, see this [code snippet](https://public.datagrok.ai/js/samples/data-access/parameterized-query). Learn how to [expose the parameter dialog to end-users as an application](../develop/how-to/create-package.md).<!--Mention?: when the cartdridge is not deployed on that particular database, the query returns an error-->
+
+:::
+
+After running a query, you typically get back a dataframe. You can perform various operations on this dataframe, such as data cleansing, transformation, [and more](../visualize/viewers/grid.md). All of these actions get recorded as macros, and will be replayed when the query is re-executed. Once you have the data set up to your satisfaction, you can add viewers and create a visualization.  
+
+To view the queries you've created or those shared with you, you can use the **Queries Gallery** (**Data** > **Queries**). This gallery provides a convenient interface to quickly access and manage queries. You can search queries by their name or tag and use the **Context Panel** to view information and actions for the selected query.
+
+## Creating views for query results
+
+You can persist the view of the query output and expose it to end users as a dashboard. For parameterized queries, these dashboards can be dynamic, allowing dashboard users to change the query parameters right inside the dashboard to view different datasets and interact with data in real-time.
```

Comment:
to view different datasets
=>
to refine your search criteria

Reply:
Also need to mention somewhere that once you execute a (parameterized) query, you can simply share the browser's URL with your colleague